### DISCUSSION TITLE
any progress in standalone app?

### DISCUSSION THREAD

**Discussion by plugeit** [flet-dev/flet#640] [Q&A] [answered]
Standalone mobile package for Flet app
We are going to investigate the way and develop a prototype for bundling together Flet framework, user program, language runtime and all dependencies into a standalone mobile package (.apk or .ipa package), so Flet program does not require a web server.

**Comment by FeodorFitsner** [comment, marked as answer]
That would be awesome!

I'm currently doing some Python integration prototypes either.
